From a30b309f1ca33656ce833733acdff6080a6864b4 Mon Sep 17 00:00:00 2001
From: Jayachandran C <jayachandranc@google.com>
Date: Thu, 18 Oct 2018 10:34:19 -0700
Subject: [PATCH 43/70] Add a new protected intent ACTION_MMSSMS_DATABASE_LOST

This intent is to catch the mmssms.db getting mysteriously lost.

Bug: 115990430
Test: Simulated mmsmsm.db corruption and verified that the intent is
broadcasted and received by application with MODIFY_PHONE_STATE
permission

Merged-in: I9a4e6d8c1548f08895a15d413356c1efc7901beb
Change-Id: I9a4e6d8c1548f08895a15d413356c1efc7901beb
---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 30760dc68..72e3cc3a2 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -38,6 +38,7 @@
     <protected-broadcast android:name="android.intent.action.ANY_DATA_STATE" />
     <protected-broadcast android:name="android.intent.action.DATA_CONNECTION_FAILED" />
     <protected-broadcast android:name="android.intent.action.DATA_STALL_DETECTED" />
+    <protected-broadcast android:name="android.intent.action.MMSSMS_DATABASE_LOST" />
     <protected-broadcast android:name="android.intent.action.SIM_STATE_CHANGED" />
     <protected-broadcast android:name="android.intent.action.NETWORK_SET_TIME" />
     <protected-broadcast android:name="android.intent.action.NETWORK_SET_TIMEZONE" />
-- 
2.17.1

